### ISSUE TITLE
check input ref, plaf, alt, in text file, the second column is a number

### ISSUE THREAD

**Issue by shajoezhu** [mcveanlab/DEploid#224]

**Comment by shajoezhu** [comment]
include test case, and bad input file for testing

**Comment by shajoezhu** [comment]
```
$ dEploid -plaf data/testData/bad.plaf.txt -vcf data/testData/PG0390-C.test.vcf -noPanel
Check input file: data/testData/bad.plaf.txt
Error: Bad conversion: Pf3D7_01_v3, int expected.
```

**Comment by shajoezhu** [comment]
closed by #225
